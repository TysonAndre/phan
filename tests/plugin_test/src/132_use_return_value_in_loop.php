<?php
for ($x = 0, is_int($x);
    sprintf("x is %d", $x),
    $x < 10;
    intdiv($x, 2)) {
    echo "X is $x\n";
}
