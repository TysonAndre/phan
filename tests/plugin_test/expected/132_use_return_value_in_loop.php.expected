src/132_use_return_value_in_loop.php:2 PhanPluginUseReturnValueInternalKnown Expected to use the return value of the internal function/method \is_int
src/132_use_return_value_in_loop.php:3 PhanPluginUseReturnValueInternalKnown Expected to use the return value of the internal function/method \sprintf
src/132_use_return_value_in_loop.php:5 PhanPluginUseReturnValueInternalKnown Expected to use the return value of the internal function/method \intdiv
